.circleci/config.yml

```diff
@@ -130,7 +130,7 @@ commands:
     steps:
       - run:
           name: Run unit tests (Pest)
-          command: php -d memory_limit=2G ./vendor/bin/pest --stop-on-failure --min=80 --coverage-clover=<< parameters.clover >>/coverage.xml --coverage-xml=<< parameters.clover >>/coverage-xml --log-junit=<< parameters.clover >>/junit.xml
+          command: php -d error_reporting='E_ALL ^ E_DEPRECATED' -d session.use_cookies=false -d session.cache_limiter=false -d memory_limit=2G ./vendor/bin/pest --stop-on-failure --min=95 --coverage-clover=<< parameters.clover >>/coverage.xml --coverage-xml=<< parameters.clover >>/coverage-xml --log-junit=<< parameters.clover >>/junit.xml
```

Comment:
This ensures test conditions are set satisfactorily, rather than having to mutate php.inis or rely on bootstrap.php conditions on the host device: pass them from the CLI directly.

Reply:
Is this line enforcing the coverage? If true, shouldn't that be enforced as part of the codecov configuration?
I noticed this repo is not configured https://about.codecov.io/language/php/

Reply:
For our PHP repos I opted to use PHP's own code coverage tooling as part of our test suites (pcov) to keep us informed on those coverage metrics, rather than relying on codecov. Just not a fan of it personally, and pcov works really smoothly for us on these PHP repos.

But this particular change is just for silencing deprecation warnings from upstream library dependencies we use that haven't fully prepared for PHP 8.1 yet; it gets very noisy unless we silence those. I'll remove them once our upstreams get updated with PHP 8.1 support and our tests are readable again without it all.